CHANGES.md

```diff
@@ -5,11 +5,21 @@
 -->
 
 <!--
-    Copyright (c) 2014, Joyent, Inc.
+    Copyright 2019 Joyent, Inc.
 -->
 
 # sdc-clients Changelog
 
+## sdc-clients 8.2.0
+
+- Backport of channel support (TRITON-886) from v12.0.0
```

Comment:
I think it could helpful to include the full changelog message from master. Perhaps:

```
- TRITON-886 Add support for `channel` option to IMGAPI client methods.
  Typically the "channel" is set as an argument to the IMGAPI client
  constructor. However, it can be useful to use an existing IMGAPI client
  to make a call for a separate channel.  (Backported from v12.0.0.)
```

Reply:
Much better, thanks.